docs/resiliency/index.asciidoc

```diff
@@ -279,7 +279,7 @@ shard will be allocated upon reopening the index.
 [float]
 === Use two phase commit for Cluster State publishing (STATUS: DONE, v5.0.0)
 
-A master node in Elasticsearch continuously https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-zen.html#fault-detection[monitors the cluster nodes]
+A master node in Elasticsearch continuously https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-fault-detection.html[monitors the cluster nodes]
```

Comment:
@lcawl  @debadair 

Based on my local doc builds, it doesn't look like we can use attributes like `{ref}` here. Let me know if that's not the case.

If not, I'll create a separate issue to include those attributes and fix up this link. I didn't want to hold this fix up in the meantime.

Reply:
I don't believe we have those attributes set up, no.

Reply:
It's just a matter of including the shared attributes...but I agree that it's not essential.